src/main/java/Main.java

```diff
@@ -130,30 +136,33 @@ public static void main(String[] args) throws Exception {
         try {
             String v = System.getProperty("java.class.version");
```

Comment:
Despite this was the choice already in place, I am actually curious what we didn't simply `java.specification.version`?

Reply:
Worked around the messy Java Specification Versions here. Also, the code in this component was using Java Class Versions originally before the Java 10+ support work started